db/migrate/20170329103706_rename_manual_record_editions_removed_document_ids_to_removed_section_ids.rb

```diff
@@ -0,0 +1,17 @@
+class RenameManualRecordEditionsRemovedDocumentIdsToRemovedSectionIds < Mongoid::Migration
+  def self.up
+    ManualRecord::Edition.collection.update(
```

Comment:
I did consider it, but given that we've already set the precedent of deleting migrations that have run in production, I decided it wasn't worth it. Also it would be easy to add if it turns out we need it.